app/src/main/java/com/github/onlynotesswent/ui/overview/Overview.kt

```diff
@@ -236,7 +236,7 @@ fun NoteItem(note: Note, onClick: () -> Unit) {
               fontWeight = FontWeight.Bold,
               color = MaterialTheme.colorScheme.onPrimaryContainer)
           Text(
-              text = note.noteClass.classCode + " " + note.noteClass.className,
+              text = note.noteClass.classCode,
```

Comment:
Maybe here having both `classCode` and `className`  was better, but I dont know if they both fit on the `noteCard`

Reply:
With the name it's too long, so for now let's just save the class code and see later if we decide to modify it